src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4720,6 +4720,21 @@ public override IOperation VisitTypeParameterObjectCreation(ITypeParameterObject
             return HandleObjectOrCollectionInitializer(operation.Initializer, initializedInstance);
         }
 
+        public override IOperation VisitDynamicObjectCreation(IDynamicObjectCreationOperation operation, int? captureIdForResult)
+        {
+            PushArray(operation.Arguments);
+            ImmutableArray<IOperation> visitedArguments = PopArray(operation.Arguments);
+
+            var hasDynamicArguments = (HasDynamicArgumentsExpression)operation;
+            IOperation initializedInstance = new DynamicObjectCreationExpression(visitedArguments, hasDynamicArguments.ArgumentNames, hasDynamicArguments.ArgumentRefKinds,
+                                                                          initializer: null, semanticModel: null, operation.Syntax, operation.Type,
```

Comment:
Bad indenting. Standard style is to indent everything 4 spaces from the start of indentation on the next line.

Reply:
> Bad indenting. Standard style is to indent everything 4 spaces from the start of indentation on the next line.

I think it is also fine to align all arguments instead, i.e. align this and the next line with beginning of ```visitedArguments``` on the previous line.

---
In reply to: [193493077](https://github.com/dotnet/roslyn/pull/27519#discussion_r193493077) [](ancestors = 193493077)

Reply:
Yup, either way is fine with me.

Reply:
Yeah, this was intended to line up. Must have accidentally hit shift+tab at some point.

---
In reply to: [193582856](https://github.com/dotnet/roslyn/pull/27519#discussion_r193582856) [](ancestors = 193582856)